lib/logstash/util/accessors.rb

```diff
@@ -55,7 +55,13 @@ def del(accessor)
 
     def include?(accessor)
       target, key = lookup_path(accessor)
-      return target.include?(key)
+      if target.is_a?(Array)
+        !target[key.to_i].nil?
+      elsif target.nil?
+        false
+      else
+        target.include?(key)
+      end
```

Comment:
I prefer to avoid too many branches and be more concise, I'd suggest:
```ruby
return false unless target
target.is_a?(Array) ? !target[key.to_i].nil? : target.include?(key)
```